### PR TITLE
Testing fixes

### PR DESCRIPTION
**Pull request by purbon**
Simple fixes for:

* Having the ci reporter used in jenkins.
* Having coveralls and simplecov installed as development dependencies.

### CODE REVIEW

File: ci/ci_setup.sh [hunk 1]
```diff
@@ -14,3 +14,7 @@ rm -rf spec/reports # no stale spec reports from previous executions
 
 # Setup the environment
 rake bootstrap # Bootstrap your logstash instance
+
+# Set up some general options for the rspec runner
+echo "--order rand" > .rspec
+echo "--format CI::Reporter::RSpec" >> .rspec
```
Comment (colinsurprenant) [inline ci/ci_setup.sh]: is there a reason for not having these options in the test tasks as in https://github.com/elasticsearch/logstash/blob/wip/bundler-move/rakelib/test.rake#L18 ?

Comment (purbon) [inline ci/ci_setup.sh]: As I said, this is only for our jenkins. If we coded in rake test:core we force everyone to use it.

Comment (purbon) [inline ci/ci_setup.sh]: I would like to keep the rake test tasks options free as much as possible, specially for the formatters, etc... but just my way of thinking.

**Comment by purbon** [comment]
@colinsurprenant tested with your fix and works, thanks with reports being generated and the PR green ;-)

File: ci/ci_setup.sh (hunk 1 above)
Comment (colinsurprenant) [inline ci/ci_setup.sh]: hmm. ok, let's get this rolling, we can iterate on that later.

**Comment by colinsurprenant** [comment]
LGTM merging in wip/bundler-move

File: ci/ci_setup.sh (hunk 1 above)
Comment (purbon) [inline ci/ci_setup.sh]: The reporters is something from our jenkins, test task should not be dependant on formatters that our Jenkins like to use ;-) ... but as I said this is my point, jejejej.

**Pull request merged**